### PR TITLE
Ops: Added expand_volume op

### PR DESCRIPTION
**Pull request by nik-redhat**
Added the expand_volume ops, and the other ops called inside it.

Fixes: #725

Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
Test run:
```
Summary:
+---------------+-------+--------------+
|    Category   | Cases | Pass Percent |
+---------------+-------+--------------+
| nonDisruptive |   1   |    100.0     |
|   Disruptive  |   0   |      0       |
|    Skipped    |   0   |      0       |
|     Total     |   1   |    100.0     |
+---------------+-------+--------------+
Table:
 vol_create_test
 
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time taken (HH:MM:SS) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:00:12       |     N/A     |
|     rep     |     PASS    |        00:00:13       |     N/A     |
|     arb     |     PASS    |        00:00:12       |     N/A     |
|     disp    |     PASS    |        00:00:11       |     N/A     |
|   dist-arb  |     PASS    |        00:00:12       |     N/A     |
|   dist-rep  |     PASS    |        00:00:13       |     N/A     |
|  dist-disp  |     PASS    |        00:00:09       |     N/A     |
+-------------+-------------+-----------------------+-------------+

 test_expand_shrink_volume
 
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time taken (HH:MM:SS) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:00:35       |     N/A     |
|     rep     |     PASS    |        00:00:38       |     N/A     |
|     arb     |     PASS    |        00:00:26       |     N/A     |
|     disp    |     PASS    |        00:00:27       |     N/A     |
|   dist-arb  |     PASS    |        00:00:27       |     N/A     |
|   dist-rep  |     PASS    |        00:00:29       |     N/A     |
|  dist-disp  |     PASS    |        00:00:25       |     N/A     |
+-------------+-------------+-----------------------+-------------+

 vol_destroy_test
 
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time taken (HH:MM:SS) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:00:23       |     N/A     |
|     rep     |     PASS    |        00:00:20       |     N/A     |
|     arb     |     PASS    |        00:00:18       |     N/A     |
|     disp    |     PASS    |        00:00:17       |     N/A     |
|   dist-rep  |     PASS    |        00:00:23       |     N/A     |
|   dist-arb  |     PASS    |        00:00:26       |     N/A     |
|  dist-disp  |     PASS    |        00:00:18       |     N/A     |
+-------------+-------------+-----------------------+-------------+



Framework runtime : 00 hours 04 minutes 24 seconds

✔ Tearing down successful.
```

File: tests/example/sample_component/test_expand_shrink_volume.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    some operations on it.
+    """
+
+    def run_test(self, redant):
+        """
+        In the testcase:
+        """
+        force = False
+        if self.volume_type == "dist-disp":
+            force = True
```
Comment (srijan-sivakumar) [inline tests/example/sample_component/test_expand_shrink_volume.py:23]: I guess this is to let glusterfs add bricks from nodes where there are bricks already present...

**Review by srijan-sivakumar — APPROVED** [review]
LGTM. Functionalities will come handy in case of sanitize volume specifically. Now that expand is there..

File: tests/example/sample_component/test_expand_shrink_volume.py (hunk 1 above)
Comment (nik-redhat) [reply tests/example/sample_component/test_expand_shrink_volume.py:23]: Yeah, specifically for dist-dip volume.

**Pull request merged**